packages/writing-style/styles/commercetools/Acronyms.yml

```diff
@@ -97,7 +97,6 @@ exceptions:
   - OR
   # Specials and Brands
   - YYYY # to be able to describe date formats like YYYY-MM-DD
-  - ZEIT # the zeit.co service is trademarked in uppercased letters.
```

Comment:
We don't need to add `Vercel` somwhere, right?

Reply:
No, there's no risk of misunderstanding with an acronym because it's not uppercased